anta/cli/get/commands.py

```diff
@@ -86,16 +87,54 @@ def from_cvp(inventory_directory: str, cvp_ip: str, cvp_username: str, cvp_passw
 @click.option(
     "--output",
     "-o",
-    default="inventory-ansible.yml",
-    help="Path to save inventory file",
+    required=False,
+    help="Path to save inventory file. If not configured, use anta inventory file",
```

Comment:
this means that we still "REQUIRE" an anta-inventory configured even when output is given - which is very weird usage .. I am not sure I like this

@mtache  thoughts?

Reply:
Agree. We need to merge #447 first.
As reported in #355, one current issue is that we require the user to define an inventory path for the command that generate this inventory: the workflow is broken.

Then, this PR can be implemented by using the inventory path as output if the user does not provide it as parameter, and fail the command if the user has not provided the output parameter nor the inventory path.

Reply:
Logic has been revamped to support PR #447 so, 

- if `--output` is defined we take it first
- if `anta --inventory` is defined and not `--output` we take it
- if `anta --inventory` is defined or not `--output` are not defined, cli fails

Also, only `--overwrite` remains. If not set and cli is running in an interactive TTY, we prompt user, else we break